collect_app/src/main/java/org/odk/collect/android/provider/FormsProvider.java

```diff
@@ -226,6 +228,12 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 
                 Timber.w("Successfully upgraded database from version %d to %d"
                         + ", without destroying all the old data", oldVersion, newVersion);
+            } else {
+                db.execSQL("ALTER TABLE " + FORMS_TABLE_NAME + " ADD COLUMN "
```

Comment:
You'll want to see whether the column already exists before adding it. See #1000 for a description of why and a solution used for the instance table.